helix-view/src/view.rs

```diff
@@ -85,10 +85,11 @@ pub struct View {
     pub last_modified_docs: [Option<DocumentId>; 2],
     /// used to store previous selections of tree-sitter objecs
     pub object_selections: Vec<Selection>,
+    pub omit_line_number_gutter: bool,
 }
 
 impl View {
-    pub fn new(doc: DocumentId) -> Self {
+    pub fn new(doc: DocumentId, omit_line_number_gutter: bool) -> Self {
```

Comment:
This should take a `gutters: Vec<>` instead

Reply:
Yes, I like your approach more, definitely better suited for the future when gutters are configurable.

Reply:
done